### DISCUSSION TITLE
.env not used when running test

### DISCUSSION THREAD

**Discussion by max13h** [nuxt/test-utils#838] [Q&A] [answered]
Hey ! I'm new to vitest and when I run a basic test I have those messages

```
Missing supabase url, set it either in `nuxt.config.js` or via env variable
Missing supabase anon key, set it either in `nuxt.config.js` or via env variable
```

Running test with nuxt context throw me errors due to .env not setup

When i `pnpm dev`, everything is setup correctly (.env is setup & nuxt.config)

I tried 
- to search on [this PR](https://github.com/nuxt/test-utils/pull/655), so I found there was a dotenv option on vitest.config like [here](https://github.com/nuxt/test-utils/blob/b783b3e02dfb7de88067a93bc953a94a6308fe9a/src/config.ts) - ([DotenvOptions](https://github.com/unjs/c12/blob/eb22c781abb9ce24e0e8b5140b2593745cb91c42/src/dotenv.ts#L5))
- use .env.test

Please if you have an idea of where to dig it will really help me 🙏

**Comment by TheAlexLichter** [comment]
Can you provide a reproduction? `.env.test` works fine in my projects ☺️

**Reply by max13h** [reply]
Thank you for you answer !
Reproduction: https://stackblitz.com/edit/nuxt-starter-1dqe4k?file=package.json
(first time i use stackblitz maybe i made mistake)

I figure out it was the empty `vitest.config.ts` that was messing with the .env
If i remove the empty config file, it works 👍 

Do you have an idea of how i can fix that ? Maybe use vitest config directly inside nuxt.config ? But never saw anything like this before...

**Reply by max13h** [reply]
I found a temporary solution by adding `dotenv.config()` to vitest.config.ts

**Reply by TheAlexLichter** [reply, marked as answer]
@max13h Use a file called `.env.test` and put your env vars in there :) Make sure they are used in your "actual application" in the same way.

**Reply by max13h** [reply]
Omg, I thought you wanted me to write my test file as my-test.nuxt.env.test.ts 😭
I've never seen .env.test before

Thank you for your help !
